bpf/lib/conntrack.h

```diff
@@ -223,6 +223,7 @@ static __always_inline __u8 __ct_lookup(const void *map, struct __ctx_buff *ctx,
 			ct_state->ifindex = entry->ifindex;
 			ct_state->dsr = entry->dsr;
 			ct_state->proxy_redirect = entry->proxy_redirect;
+			/* See the ct_create4 comments re the rx_bytes hack */
 			if (dir == CT_SERVICE)
```

Comment:
@brb I just became aware of this hack. Would it be worth writing a small Coccinelle script to check that all assignments of `x != entry->rx_bytes` to `ct_state->backend_id` are guarded by a check that `dir == z; z != CT_SERVICE`? Happy to write it given it's fairly trivial. It would guarantee we have a test failing if the assumption behind this hack ever becomes false.

Reply:
Yeah, makes sense. Thanks!